processing/src/main/java/org/apache/druid/query/rowsandcols/concrete/FrameRowsAndColumns.java

```diff
@@ -91,4 +93,10 @@ public <T> T as(Class<T> clazz)
     }
     return null;
   }
+
+  @Override
+  public void close() throws IOException
+  {
+    // nothing to close
```

Comment:
I am wondering if we need to clear the cache here. @LakshSingla - maybe you know?

Reply:
I think since the close is called, there'd be no attempt to use the object, so we don't wanna clear the cache. 
I was wondering if there's a .close() method on the cached objects or any other resources that we are seeing, but I don't think there is.